app/scripts/directives/oscPersistentVolumeClaim.js

```diff
@@ -22,8 +23,18 @@ angular.module("openshiftConsole")
         }, {
           value: "Pi",
           label: "PiB"
-        }];
+          }];
         scope.claim.selectedLabels = [];
+        DataService.get({
+          group: 'storage.k8s.io',
+          resource: 'storageclasses'
+        }, null, {}).then(
```

Comment:
Also you should not be using DataService.get here, you need to use DataService.list and then take out the second null parameter